packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/components/steps/link-user-archive/controller.js

```diff
@@ -235,7 +239,7 @@ export default class ColdArchiveLinkUserArchiveController {
     this.userModel.linkedMode.credential = null;
   }
 
-  onLinkedUserClicked(user) {
+  async onLinkedUserClicked(user) {
```

Comment:
Since there is no await is async necessary ?

Reply:
Agree with @SimonChaumet , to remove